libosinfo.rb

```diff
@@ -0,0 +1,91 @@
+class Libosinfo < Formula
+  desc "The Operating System information database"
+  homepage "https://libosinfo.org/"
+  url "https://fedorahosted.org/releases/l/i/libosinfo/libosinfo-0.3.0.tar.gz"
+  sha256 "538a3468792e919edf5364fe102d751353ae600a92ad3a24f024424a182cefbc"
+
+  depends_on "intltool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "wget" => :build
+
```

Comment:
Style nit but I'm kind of "bleh" about leaving spacing between dep blocks unless there's `if build.with?` involved.